recipes/google-guetzli/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GoogleGuetzliConan(ConanFile):
+    name = "google-guetzli"
```

Comment:
```suggestion
    name = "guetzli"
```

Change the name to guetzli only, don't forget to update the folder name too.